src/sources/aws_ecs_metrics/mod.rs

```diff
@@ -86,8 +93,8 @@ pub struct AwsEcsMetricsSourceConfig {
     namespace: String,
 }
 
-const METADATA_URI_V4: &str = "ECS_CONTAINER_METADATA_URI";
-const METADATA_URI_V3: &str = "ECS_CONTAINER_METADATA_URI_V4";
```

Comment:
:scream:

Reply:
Ah, I meant to add a note about that https://github.com/vectordotdev/vector/issues/12917

Reply:
Actually, I'm going to back this out - doesn't feel nice throwing the change in here.

Reverted in 9c116e685